### PR TITLE
IfcFlowInstrument Rail Import BIMQ 2021-07-21T07:00:01.4679226Z

### PR DESCRIPTION
**Pull request by jmirtsch**
Rail properties for IfcFlowInstrument

### CODE REVIEW

**Review by AlexBrad1eyCT — CHANGES_REQUESTED** [review]
See PSet comments i think this is in the wrong place completely

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_FlowInstrumentTypeElectricalCombination/DocPropertySet.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+		<DocProperty xsi:nil="true" href="RatedVoltage_0KkDpffYz8L86gzG31TCSi" />
+		<DocProperty xsi:nil="true" href="PhaseNumber_1Lt0BWQy1AD811dqTjGf_d" />
+		<DocProperty xsi:nil="true" href="PrimaryCurrent_3exZs5G_1BZR8ePjN8RlDv" />
+		<DocProperty xsi:nil="true" href="PrimaryFrequency_3bDvHjEfEAwV9_ElCeTdI" />
+		<DocProperty xsi:nil="true" href="PrimaryVoltage_2vDDOIIxP0uSbVrMQcOKz" />
+		<DocProperty xsi:nil="true" href="SecondaryCurrent_0BPcq9v3b1rOX4CuGa6OPK" />
+		<DocProperty xsi:nil="true" href="SecondaryFrequency_0AQIcoro9EcAvh1YvLImld" />
+		<DocProperty xsi:nil="true" href="SecondaryVoltage_0jvGIYxLz90vG7yrY_qQ8T" />
+	</Properties>
+</DocPropertySet>
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_FlowInstrumentTypeElectricalCombination/DocPropertySet.xml]: I have splited the properties into the 3 individuals pset to make it more understandable. 
I changed definition and naming of the Pset to make them more suitable. da44e1e809cdc1b4a54b066283bc2ff060546734

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_FlowInstrumentTypeElectricalCombination/Documentation.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Properties of Combined Instrument Transformer used in railway. The property set can be used by the predefined type COMBINATION of IfcFlowInstrument
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_FlowInstrumentTypeElectricalCombination/Documentation.md]: Same as above. Main function is mettering. da44e1e809cdc1b4a54b066283bc2ff060546734

File: IFC4x3/Properties/p/PrimaryFrequency_3bDvH$jEfEAwV9_ElCeTdI/Documentation.md [hunk 3]
```diff
@@ -0,0 +1 @@
+The frequency that is going to be transformed and that runs into the transformer on the primary side.
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Properties/p/PrimaryFrequency_3bDvH$jEfEAwV9_ElCeTdI/Documentation.md:1]: I have reused Primary frequency from transformer. da44e1e809cdc1b4a54b066283bc2ff060546734

File: IFC4x3/Properties/s/SecondaryFrequency_0AQIcoro9EcAvh1YvLImld/Documentation.md [hunk 4]
```diff
@@ -0,0 +1 @@
+The frequency that has been transformed and is running out of the transformer on the secondary side.
```
[the remark this answers is not in the document]
Comment (fhulin) [reply IFC4x3/Properties/s/SecondaryFrequency_0AQIcoro9EcAvh1YvLImld/Documentation.md]: In fact the main function of instrument transformer is to operate instruments or metering from high voltage or high current circuits. There is no energy transformation. So we decided to put it in flow instrument more than transformer. da44e1e809cdc1b4a54b066283bc2ff060546734

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_VoltageInstrumentTransformer/DocPropertySet.xml [hunk 5]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_VoltageInstrumentTransformer" UniqueId="4b47169f-04e8-f329-e589-ca2bfe243b09" ApplicableType="IfcFlowInstrument/VOLTMETER" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="SecondaryVoltage_0jvGIYxLz90vG7yrY_qQ8T" />
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_VoltageInstrumentTransformer/DocPropertySet.xml]: Forgot to removed deleted property, so is duplicated in PSet

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CurrentInstrumentTransformer/DocPropertySet.xml [hunk 6]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_CurrentInstrumentTransformer" UniqueId="ba0ea724-fc32-6d71-7b15-cc2475522467" ApplicableType="IfcFlowInstrument/AMMETER" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="SecondaryCurrent_0BPcq9v3b1rOX4CuGa6OPK" />
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CurrentInstrumentTransformer/DocPropertySet.xml]: remove this duplicated property and the definition

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_VoltageInstrumentTransformer/DocPropertySet.xml [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+		<DocProperty xsi:nil="true" href="AccuracyClass_32YEer4tP9EB633g5FK49p" />
+		<DocProperty xsi:nil="true" href="AccuracyGrade_3y6bNc8F56Hexr9vdk2L" />
+		<DocProperty xsi:nil="true" href="RatedVoltage_1uw9k0qSWHuO00025QrEV" />
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_VoltageInstrumentTransformer/DocPropertySet.xml]: This is on Pset_ElectricalDeviceCommon, suggestion: remove
if this refers to the instruments operating power supply voltage

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_FlowInstrumentTypeVoltmeter/DocPropertySet.xml [hunk 8]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_FlowInstrumentTypeVoltmeter" UniqueId="4b47169f-04e8-f329-e589-ca2bfe243b09" ApplicableType="IfcFlowInstrument/VOLTMETER" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="SecondaryVoltage_0jvGIYxLz90vG7yrY_qQ8T" />
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_FlowInstrumentTypeVoltmeter/DocPropertySet.xml]: this is the wrong secondaryVoltage
```suggestion
		<DocProperty xsi:nil="true" href="SecondaryVoltage_1ZIvG0qX4HuO00025QrEV" />
```

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_FlowInstrumentTypeElectricalCombination/DocPropertySet.xml [hunk 9]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_FlowInstrumentTypeElectricalCombination" UniqueId="8f68a974-45e4-30c7-5db5-c91cafca08a3" ApplicableType="IfcFlowInstrument/COMBINED" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="AccuracyClass_32YEer4tP9EB633g5FK49p" />
+		<DocProperty xsi:nil="true" href="AccuracyGrade_3y6bNc8F56Hexr9vdk2L" />
+		<DocProperty xsi:nil="true" href="ApparentPower_1n4pozW90Wu_Sinh6vpNI" />
+		<DocProperty xsi:nil="true" href="NominalCurrent_1Sia_y4DvB595bUBLM1Dsb" />
+		<DocProperty xsi:nil="true" href="NominalPower_2pu16hiH1CqQv04BNgxAph" />
+		<DocProperty xsi:nil="true" href="RatedVoltage_0KkDpffYz8L86gzG31TCSi" />
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_FlowInstrumentTypeElectricalCombination/DocPropertySet.xml]: same as other comment on ratedVoltage

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_FlowInstrumentTypeElectricalCombination/DocPropertySet.xml [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+		<DocProperty xsi:nil="true" href="PhaseNumber_1Lt0BWQy1AD811dqTjGf_d" />
+		<DocProperty xsi:nil="true" href="PrimaryCurrent_3exZs5G_1BZR8ePjN8RlDv" />
+		<DocProperty xsi:nil="true" href="PrimaryFrequency_3bDvHjEfEAwV9_ElCeTdI" />
+		<DocProperty xsi:nil="true" href="PrimaryVoltage_2vDDOIIxP0uSbVrMQcOKz" />
+		<DocProperty xsi:nil="true" href="SecondaryCurrent_0BPcq9v3b1rOX4CuGa6OPK" />
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_FlowInstrumentTypeElectricalCombination/DocPropertySet.xml]: using different versions of secondary current this is the created one

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CurrentInstrumentTransformer/DocPropertySet.xml [hunk 11]
```diff
@@ -0,0 +1,17 @@
...
+		<DocProperty xsi:nil="true" href="AccuracyClass_32YEer4tP9EB633g5FK49p" />
+		<DocProperty xsi:nil="true" href="AccuracyGrade_3y6bNc8F56Hexr9vdk2L" />
+		<DocProperty xsi:nil="true" href="RatedVoltage_1uw9k0qSWHuO00025QrEV" />
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CurrentInstrumentTransformer/DocPropertySet.xml]: same as other rated Voltage comment

File: IFC4x3/Properties/p/PhaseNumber_1Lt0BWQy1AD811dqTjGf_d/Documentation.md [hunk 12]
```diff
@@ -0,0 +1 @@
+Number of phase that the equipment operates on.
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Properties/p/PhaseNumber_1Lt0BWQy1AD811dqTjGf_d/Documentation.md]: this has a suggestion on another pull request needs to be harmonised.

File: IFC4x3/Properties/a/ApparentPower_1n4$pozW90Wu_Sinh6vpNI/DocProperty.xml [hunk 13]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="ApparentPower_1n4pozW90Wu_Sinh6vpNI" Name="ApparentPower" UniqueId="7113fcf2-f602-4083-8f9c-b31ac6e735d2" PrimaryDataType="IfcPowerMeasure" />
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Properties/a/ApparentPower_1n4$pozW90Wu_Sinh6vpNI/DocProperty.xml]: Can you clarify the difference between NominalPower and ApparentPower as the description on nominal power says: Apparent Power in it.

**Review by AlexBrad1eyCT — CHANGES_REQUESTED** [review]
@fhulin, can you look at the currently proposed PSets and check they are all needed, they seem quite duplicated and there is a set named classically PSet_FlowInstrumentTypeXXX and a set with Transformer. is this a mistake or on purpose?

- inconsitant property use
- duplicates in PSets.

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_VoltageInstrumentTransformer/DocPropertySet.xml (hunk 5 above)
Comment (fhulin) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_VoltageInstrumentTransformer/DocPropertySet.xml]: Done. Commit : 0ed68dc80cfee0e7a216fa8ce063c47b9d42fe5f

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CurrentInstrumentTransformer/DocPropertySet.xml (hunk 6 above)
Comment (fhulin) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CurrentInstrumentTransformer/DocPropertySet.xml]: Done. Commit : 0ed68dc80cfee0e7a216fa8ce063c47b9d42fe5f 
What do you mean by def?

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_FlowInstrumentTypeVoltmeter/DocPropertySet.xml (hunk 8 above)
Comment (fhulin) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_FlowInstrumentTypeVoltmeter/DocPropertySet.xml]: 0ed68dc80cfee0e7a216fa8ce063c47b9d42fe5f

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_FlowInstrumentTypeElectricalCombination/DocPropertySet.xml (hunk 10 above)
Comment (fhulin) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_FlowInstrumentTypeElectricalCombination/DocPropertySet.xml]: Done. Commit : 0ed68dc80cfee0e7a216fa8ce063c47b9d42fe5f

Comment (AlexBrad1eyCT) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_FlowInstrumentTypeElectricalCombination/DocPropertySet.xml]: You missed it... :smile:

File: IFC4x3/Properties/a/AccuracyGrade_3y6bNc8F56Hexr9v$dk$2L/Documentation.md [hunk 14]
```diff
@@ -0,0 +1 @@
+The grade of accuracy
```
Comment (SergejMuhic) [inline IFC4x3/Properties/a/AccuracyGrade_3y6bNc8F56Hexr9v$dk$2L/Documentation.md]:
```suggestion
The grade of accuracy.
```

File: IFC4x3/Properties/a/ApparentPower_1n4$pozW90Wu_Sinh6vpNI/Documentation.md [hunk 15]
```diff
@@ -0,0 +1 @@
+The apparent power of the equipment
```
Comment (SergejMuhic) [inline IFC4x3/Properties/a/ApparentPower_1n4$pozW90Wu_Sinh6vpNI/Documentation.md]:
```suggestion
The apparent power of the equipment.
```

File: IFC4x3/Properties/a/ApparentPower_1n4$pozW90Wu_Sinh6vpNI/DocProperty.xml (hunk 13 above)
Comment (SergejMuhic) [reply IFC4x3/Properties/a/ApparentPower_1n4$pozW90Wu_Sinh6vpNI/DocProperty.xml]: Second this.

File: IFC4x3/Properties/n/NominalCurrent_1Sia_y4DvB595bUBLM1Dsb/DocProperty.xml [hunk 16]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NominalCurrent_1Sia_y4DvB595bUBLM1Dsb" Name="NominalCurrent" UniqueId="5cb24fbc-10de-4b14-9165-78b55604dda5" PrimaryDataType="IfcElectricCurrentMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/n/NominalCurrent_1Sia_y4DvB595bUBLM1Dsb/DocProperty.xml]: Reuse.

File: IFC4x3/Properties/n/NominalPower_2pu16hiH1CqQv04BNgxAph/DocProperty.xml [hunk 17]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NominalPower_2pu16hiH1CqQv04BNgxAph" Name="NominalPower" UniqueId="b3e011ab-b110-4cd1-ae40-10b5eaecaceb" PrimaryDataType="IfcPowerMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/n/NominalPower_2pu16hiH1CqQv04BNgxAph/DocProperty.xml]: Reuse.

File: IFC4x3/Properties/p/PhaseNumber_1Lt0BWQy1AD811dqTjGf_d/DocProperty.xml [hunk 18]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PhaseNumber_1Lt0BWQy1AD811dqTjGf_d" Name="PhaseNumber" UniqueId="55dc02e0-6bc0-4a34-8041-9f476d429fa7" PrimaryDataType="IfcIdentifier" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/p/PhaseNumber_1Lt0BWQy1AD811dqTjGf_d/DocProperty.xml]: Seen this. Harmonise.

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CombinationInstrumentTransformer/Documentation.md [hunk 19]
```diff
@@ -0,0 +1 @@
+Instrument transformers are high accuracy class electrical devices used to isolate or transform voltage or current levels. The main function of instrument transformer is to operate instruments or metering from high voltage or high current circuits, safely isolating secondary control circuitry from the high voltages or currents. Combination Instrument transformer is metering current and voltage.
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CombinationInstrumentTransformer/Documentation.md]:
```suggestion
Instrument transformers are high accuracy class electrical devices used to isolate or transform voltage or current levels. The main function of instrument transformers is to operate instruments or metering from high voltage or high current circuits, safely isolating secondary control circuitry from the high voltages or currents. Combination instrument transformers are metering current and voltage.
```

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CurrentInstrumentTransformer/Documentation.md [hunk 20]
```diff
@@ -0,0 +1 @@
+Instrument transformers are high accuracy class electrical devices used to isolate or transform voltage or current levels. The main function of instrument transformer is to operate instruments or metering from high voltage or high current circuits, safely isolating secondary control circuitry from the high voltages or currents. Combination Instrument transformer is metering current.
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CurrentInstrumentTransformer/Documentation.md]:
```suggestion
Instrument transformers are high accuracy class electrical devices used to isolate or transform voltage or current levels. The main function of instrument transformers is to operate instruments or metering from high voltage or high current circuits, safely isolating secondary control circuitry from the high voltages or currents. Combination instrument transformers are metering current.
```

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_FlowInstrumentTypeAmpermeter/Documentation.md [hunk 21]
```diff
@@ -0,0 +1 @@
+Properties of Current Instrument Transformer used in railway. The property set can be used by the predefined type AMPERMETER of IfcFlowInstrument
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_FlowInstrumentTypeAmpermeter/Documentation.md]:
```suggestion
Properties of current instrument transformer used in railway. The property set can be used by the predefined type AMPERMETER of IfcFlowInstrument,
```

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_FlowInstrumentTypeElectricalCombination/Documentation.md (hunk 2 above)
Comment (SergejMuhic) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_FlowInstrumentTypeElectricalCombination/Documentation.md]:
```suggestion
Properties of combined instrument transformer used in railway. The property set can be used by the predefined type COMBINATION of IfcFlowInstrument.
```

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_VoltageInstrumentTransformer/Documentation.md [hunk 22]
```diff
@@ -0,0 +1 @@
+Instrument transformers are high accuracy class electrical devices used to isolate or transform voltage or current levels. The main function of instrument transformer is to operate instruments or metering from high voltage or high current circuits, safely isolating secondary control circuitry from the high voltages or currents. Combination Instrument transformer is metering voltage.
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_VoltageInstrumentTransformer/Documentation.md]:
```suggestion
Instrument transformers are high accuracy class electrical devices used to isolate or transform voltage or current levels. The main function of instrument transformers is to operate instruments or metering from high voltage or high current circuits, safely isolating secondary control circuitry from the high voltages or currents. Combination instrument transformers are metering voltage.
```

**Review by SergejMuhic — COMMENTED** [review]
Pset descriptions are awfully similar. Intended?

File: IFC4x3/Properties/a/ApparentPower_1n4$pozW90Wu_Sinh6vpNI/DocProperty.xml (hunk 13 above)
Comment (fhulin) [reply IFC4x3/Properties/a/ApparentPower_1n4$pozW90Wu_Sinh6vpNI/DocProperty.xml]: Concept are realy close, so I have erased apparent power.

File: IFC4x3/Properties/n/NominalPower_2pu16hiH1CqQv04BNgxAph/DocProperty.xml (hunk 17 above)
Comment (fhulin) [reply IFC4x3/Properties/n/NominalPower_2pu16hiH1CqQv04BNgxAph/DocProperty.xml]: Already reused

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_VoltageInstrumentTransformer/DocPropertySet.xml (hunk 7 above)
Comment (fhulin) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_VoltageInstrumentTransformer/DocPropertySet.xml]: Just discussed!

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CurrentInstrumentTransformer/DocPropertySet.xml (hunk 11 above)
Comment (fhulin) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CurrentInstrumentTransformer/DocPropertySet.xml]: Done Commit : 6be975f2f1a62ccae3a2de607523c16a192995d1

File: IFC4x3/Properties/p/PhaseNumber_1Lt0BWQy1AD811dqTjGf_d/Documentation.md (hunk 12 above)
Comment (fhulin) [reply IFC4x3/Properties/p/PhaseNumber_1Lt0BWQy1AD811dqTjGf_d/Documentation.md]: Done. Commit : 6be975f2f1a62ccae3a2de607523c16a192995d1

File: IFC4x3/Properties/p/PhaseNumber_1Lt0BWQy1AD811dqTjGf_d/DocProperty.xml (hunk 18 above)
Comment (fhulin) [reply IFC4x3/Properties/p/PhaseNumber_1Lt0BWQy1AD811dqTjGf_d/DocProperty.xml]: Done. Commit : 6be975f2f1a62ccae3a2de607523c16a192995d1

File: IFC4x3/Properties/n/NominalCurrent_1Sia_y4DvB595bUBLM1Dsb/DocProperty.xml (hunk 16 above)
Comment (fhulin) [reply IFC4x3/Properties/n/NominalCurrent_1Sia_y4DvB595bUBLM1Dsb/DocProperty.xml]: 6be975f2f1a62ccae3a2de607523c16a192995d1

File: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CombinationInstrumentTransformer/DocPropertySet.xml [hunk 23]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_CombinationInstrumentTransformer" UniqueId="8f68a974-45e4-30c7-5db5-c91cafca08a3" ApplicableType="IfcFlowInstrument/COMBINED" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CombinationInstrumentTransformer/DocPropertySet.xml]: Do we need the combination Pset or can we just add the voltage and current ones to a combined PSet.

Comment (SergejMuhic) [reply IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CombinationInstrumentTransformer/DocPropertySet.xml]: As stated in my comment, these are awfully similar.

**Review by AlexBrad1eyCT — APPROVED** [review]
All Good

**Review by SergejMuhic — APPROVED** [review]

**Pull request merged**